src/Composer/Installer.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\ManagerPlugin\Composer;
+
+use Composer\Package\Locker;
+use Composer\Package\RootPackageInterface;
+use Contao\ManagerPlugin\Dependency\DependencyResolverTrait;
+use Contao\ManagerPlugin\Dependency\DependentPluginInterface;
+use Contao\ManagerPlugin\Dependency\UnresolvableDependenciesException;
+
+class Installer
+{
+    use DependencyResolverTrait;
+
+    private static $generatedClassTemplate = <<<'PHP'
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
```

Comment:
We should probably use `date('Y')` here.

Reply:
Fixed in 844d0c7ae8248c966659f5864e45c9e56dfe185e